src/pages/publication/references_tab/BookReferenceForm.tsx

```diff
@@ -6,7 +6,7 @@ import styled from 'styled-components';
 import { Checkbox, FormControlLabel, MenuItem, TextField } from '@material-ui/core';
 
 import { Publication } from '../../../types/publication.types';
-import { BookFieldNames, bookTypes, emptyPublisher } from '../../../types/references.types';
+import { BookFieldNames, BookType, emptyPublisher } from '../../../types/references.types';
```

Comment:
endret til singular på de enum jeg var innom (https://stackoverflow.com/questions/1335959/singular-or-plural-for-enumerations/1336012)